HelpSource/Classes/SimpleNumber.schelp

```diff
@@ -974,51 +974,55 @@ stores this on the given stream
 method:: printOn
 prints this on the given stream
 
+
 subsection:: random
+Unless noted otherwise, the return value of all these methods will be an link::Classes/Integer:: only when the receiver and all arguments are Integers as well.
 
 method:: coin
 Let emphasis::x:: be the receiver clipped to the range [0, 1]. With probability emphasis::x::, return true. With probability 1 - emphasis::x::, return false.
 
 method:: rand
-returns:: Random number from zero up to the receiver, exclusive.
+Random number uniformly distributed in the range [0, this).
 
 method:: rand2
-returns:: a random number from -this to +this.
+Random number uniformly distributed in the range [-this, +this].
 
 method:: rrand
-argument::aNumber
-the upper limit
+Random number uniformly distributed in the range [this, hi]. Note that unlike link::Classes/SimpleNumber#-rand::, the range is inclusive of the upper bound.
+argument::hi
+The upper limit of the range.
 argument::adverb
-returns:: a random number in the interval ]a, b[.
-discussion::
-If both a and b are link::Classes/Integer:: then the result will be an link::Classes/Integer::.
+discussion:: Equivalent to code::(hi - lo) * 1.0.rand + lo::, except for the inclusive upper bound.
 
 method:: linrand
-returns:: a linearly distributed random number from zero to this.
+Random number linearly distributed in the range [0, this], i.e., sampled from a distribution over [0, 1] with a probability density function (PDF) of 2(1 - x) and scaled by this.
 
 method:: bilinrand
-returns:: Bilateral linearly distributed random number from -this to +this.
+Bilateral linearly distributed random number in the range [-this, +this]. Like linrand, but the PDF is also supported in the range [-1, 0] by 2(x + 1).
 
 method:: sum3rand
-This was suggested by Larry Polansky as a loose approximation of gaussian.
-returns:: A random number from -this to +this that is the result of summing three uniform random generators to yield a bell-like distribution.
+Random number from -this to +this that is the result of summing three uniform random generators to yield a bell-like distribution.
```

Comment:
you updated other method descriptions to use "in the range [-this, +this]" instead of "from -this to +this", so good to use it here for consistency.